bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros-protocol.html" import feature_card with context %}
+{% from "macros.html" import fxa_email_form, google_play_button, fxa_create_account_button with context %}
+
+{% extends "base-protocol.html" %}
+
+{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
+{% block page_desc %}{{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('firefox_accounts_2018') }}
+{% endblock %}
+
+{% block content %}
+
+<section class="mzp-c-hero mzp-has-image mzp-t-firefox mzp-t-product-firefox">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('One log-in. Power and privacy everywhere.') }}</h1>
+      <div id="hero-cta" class="mzp-c-hero-cta">
+        <div class="show-fxa-supported-signed-out">
+          <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-button-light&utm_content=%2Ffirefox%2Faccounts%2F&utm_campaign=Create an Account" class="button account-benefits" data-button-name="Create account" data-link-type="button" data-cta-position="primary cta" id="features-hero-account">
+            {{ _('Create a Firefox Account') }}
+          </a>
+        </div>
+        <div class="show-fxa-supported-signed-in">
+          <button class="mzp-c-button" type="button" id="fx-modal">{{ _('Download the Firefox App') }}</button>
+        </div>
+        <div id="features-hero-download" class="show-fxa-not-fx">
```

Comment:
You're using this ID to target the functional test, but it should actually be on the button itself and not the wrapper div.

```
{{ download_firefox(dom_id='features-hero-download', download_location='primary cta') }}
```